onnx/defs/logical/defs.cc

```diff
@@ -168,11 +172,19 @@ Bitwise shift operator performs element-wise operation. For each input element,
  not necessarily identical.
 )DOC";
 
+static std::string GetBitShiftDoc() {
+#ifndef __ONNX_NO_DOC_STRINGS
+  return std::string(BitShift_ver11_doc) + GenerateBroadcastingDocMul();
+#else
+  return "";
+#endif
+}
+
 ONNX_OPERATOR_SET_SCHEMA(
     BitShift,
     11,
     OpSchema()
-        .SetDoc(std::string(BitShift_ver11_doc) + GenerateBroadcastingDocMul())
+        .SetDoc(GetBitShiftDoc())
```

Comment:
The addition of the output from GenerateBroadcastingDocMul() inhibits the linker from throwing away BitShift_ver11_doc. Hence the introduction of the function GetBitShiftDoc().